deploy/examples/subvolumegroup.yaml

```
@@ -6,6 +6,14 @@ metadata:
   namespace: rook-ceph # namespace:cluster
 spec:
   # The name of the subvolume group. If not set, the default is the name of the subvolumeGroup CR.
-  # name: group-a
+  # name: group-a1
   # filesystemName is the metadata name of the CephFilesystem CR where the subvolume group will be created
   filesystemName: myfs
+  # reference https://docs.ceph.com/en/latest/cephfs/fs-volumes/#pinning-subvolumes-and-subvolume-groups
+  # only one out of (export, distributed, random) can be set at a time
+  # by default pinning is set with default values(distributed=1)
+  # for disabling default values set(distributed=0)
+  pinning:
+    distributed: 1          # distributed=<0, 1>(disabled=0)
+    #export:                 # export=<0-256>(disabled=-1)
+    #random:                 # random=[0.0, 1.0](disabled=0.0)
```

I'm having trouble understanding this syntax, and I think it is because the parenthetical is attached to the rest, which makes it appear to be some kind of `map<key>(value)` syntax. Also, we should follow a consistent pattern of commenting-out values with `# ` including the space.

```suggestion
    distributed: 1          # distributed=<0, 1> (disabled=0)
    # export:                 # export=<0-256> (disabled=-1)
    # random:                 # random=[0.0, 1.0] (disabled=0.0)
```